### PR TITLE
Fix #65890: Update the bot message to use github pull requests

### LINKED ISSUE #65890: Good first issue bot instructs to create a review on phabricator
When an issue is tagged 'good first issue', the automatic comment bot instruct people to create a review on phabricator.
This should be updated

Ex https://github.com/llvm/llvm-project/issues/65858#issuecomment-1712784751

### PR DESCRIPTION
**Pull request by da-viper**
Fixes #65890

### CODE REVIEW

**Review by tru — APPROVED** [review]
LGTM

File: llvm/utils/git/github-automation.py [hunk 1]
```diff
@@ -23,17 +23,15 @@
 
 This issue may be a good introductory issue for people new to working on LLVM. If you would like to work on this issue, your first steps are:
 
-  1) Assign the issue to you.
+  1) In the comments of the issue, request for it to be assigned to you.
   2) Fix the issue locally.
   3) [Run the test suite](https://llvm.org/docs/TestingGuide.html#unit-and-regression-tests) locally.
     3.1) Remember that the subdirectories under `test/` create fine-grained testing targets, so you can
          e.g. use `make check-clang-ast` to only run Clang's AST tests.
   4) Create a `git` commit
```
Comment (SimplyDanny) [inline llvm/utils/git/github-automation.py:31]:
```suggestion
  4) Create a Git commit.
```

File: llvm/utils/git/github-automation.py [hunk 2]
```diff
@@ -23,17 +23,15 @@
...
   5) Run [`git clang-format HEAD~1`](https://clang.llvm.org/docs/ClangFormat.html#git-integration) to format your changes.
-  6) Submit the patch to [Phabricator](https://reviews.llvm.org/).
-    6.1) Detailed instructions can be found [here](https://llvm.org/docs/Phabricator.html#requesting-a-review-via-the-web-interface)
-
-For more instructions on how to submit a patch to LLVM, see our [documentation](https://llvm.org/docs/Contributing.html).
+  6) Submit the pull request in the [github repository](https://github.com/llvm/llvm-project/pulls).
```
Comment (SimplyDanny) [inline llvm/utils/git/github-automation.py:33]:
```suggestion
  6) Open a pull request to the [upstream repository](https://github.com/llvm/llvm-project/pulls) on GitHub.
```

File: llvm/utils/git/github-automation.py [hunk 3]
```diff
@@ -23,17 +23,15 @@
...
+    6.1) Detailed instructions can be found [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
```
Comment (SimplyDanny) [inline llvm/utils/git/github-automation.py:34]:
```suggestion
    6.1) Detailed instructions can be found [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
```

**Review by SimplyDanny — COMMENTED** [review]
Proposing a few smaller improvements:

File: llvm/utils/git/github-automation.py [hunk 4]
```diff
@@ -23,17 +23,15 @@
...
     3.1) Remember that the subdirectories under `test/` create fine-grained testing targets, so you can
          e.g. use `make check-clang-ast` to only run Clang's AST tests.
-  4) Create a `git` commit
+  4) Create a Git commit.
   5) Run [`git clang-format HEAD~1`](https://clang.llvm.org/docs/ClangFormat.html#git-integration) to format your changes.
-  6) Submit the patch to [Phabricator](https://reviews.llvm.org/).
-    6.1) Detailed instructions can be found [here](https://llvm.org/docs/Phabricator.html#requesting-a-review-via-the-web-interface)
-
-For more instructions on how to submit a patch to LLVM, see our [documentation](https://llvm.org/docs/Contributing.html).
+  6) Open a pull request to the [upstream repository](https://github.com/llvm/llvm-project/pulls) on GitHub.
```
Comment (SimplyDanny) [inline llvm/utils/git/github-automation.py:33]:
```suggestion
  6) Open a [pull request](https://github.com/llvm/llvm-project/pulls) to the [upstream repository](https://github.com/llvm/llvm-project) on GitHub.
```

**Review by SimplyDanny — APPROVED** [review]

**Review by JOE1994 — APPROVED** [review]

**Pull request merged**